backend/routes/group.py

```
@@ -0,0 +1,53 @@
+
+router = APIRouter()
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+@router.get("/{university_name}", response_model=UniversityPage)
+def get_university_info(university_name: str, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_university_info`

(ANN201)